backend/capellacollab/projects/capellamodels/models.py

```diff
@@ -31,39 +38,96 @@ class CapellaModelType(enum.Enum):
     LIBRARY = "library"
 
 
-class DB_CapellaModel(Base):
-    __tablename__ = "capella_models"
+class NewModel(BaseModel):
```

Comment:
As it is not just a new model (the attributes include also already created models), I'd rename it to just `CapellaModel` instead of `NewModel`.